src/OrchardCore.Modules/OrchardCore.AuditTrail/Models/AuditTrailEvent.cs

```diff
@@ -0,0 +1,58 @@
+﻿using OrchardCore.Entities;
+using System;
+
+namespace OrchardCore.AuditTrail.Models
+{
+    public class AuditTrailEvent : Entity
+    {
+        /// <summary>
+        /// The ID of the event.
+        /// </summary>
+        public string Id { get; set; }
```

Comment:
This can't be called `Id` as I see you have a partial implement with an IdGenerator.

It needs to be called `AuditTrailEventId` or it will conflict with the `YesSql` `Id`